src/components/popover-container/popover-container.component.tsx

```diff
@@ -25,6 +25,8 @@ export interface RenderOpenProps {
   ref: React.RefObject<HTMLButtonElement>;
   "aria-label"?: string;
   id?: string;
+  "aria-expanded": boolean;
+  "aria-haspopup": boolean;
```

Comment:
Do we want to restrict this to just boolean when the value could also be a string like `menu`, `dialog` or `listbox`? 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-haspopup#values